util/Migrator/DbMigrator.cs

```diff
@@ -130,6 +132,17 @@ public DbMigrator(string connectionString, ILogger<DbMigrator> logger)
         }
 
         var upgrader = builder.Build();
+
+        if (dryRun)
+        {
+            var scriptsToExec = upgrader.GetScriptsToExecute();
+            foreach (var script in scriptsToExec)
+            {
+                _logger.LogInformation(Constants.BypassFiltersEventId, script.Name);
+            }
```

Comment:
🎨 Let's not use a log statement for every script. Instead, create a `StringBuilder` earlier and append to it here, while also including `Environment.Newline` at the end. This will render much better.

Reply:
I used string builder here.